account-kit/smart-contracts/src/6900-v08-RI/account/account.ts

```diff
@@ -0,0 +1,121 @@
+import type {
```

Comment:
Do we need RI in the file path?

Reply:
I think it'd be helpful to have separate entries in the SDK by account implementation, and both the RI and other production account implementations will support 6900 v0.8. Maybe we could name it just `6900-RI`?

Reply:
I don't think we will have SDK for both RI and MAv2 (or some other implementation). I do like the v0.8 wording since we might have a v0.9 in the future.

Reply:
Why not have both the RI and MAv2? They're different account implementations

Reply:
Would the interfaces diff between them? RI's interfaces is very likely a subset of MA v2's, no?

Reply:
Synced with client, aa-sdk will only support mav2, wont support both.